### DISCUSSION TITLE
How to avoid wire:initial-data JSON

### DISCUSSION THREAD

**Discussion by magarrent** [livewire/livewire#5100] [1. Help]
Hey there, is there any way to NOT send all the data via `wire:initial-data` attribute?

I have an extensive list of objects from external API, and I think that's not the best way to send the data to the view.

Any ideas? Is that possible? Thanks!

**Comment by joshhanley** [comment]
@magarrent you could not load the data into a public property on the initial request, and instead load the data using `wire:init` which will send the data to the front end as part of the json request. See https://laravel-livewire.com/docs/2.x/defer-loading

Hope this helps!
